Package.swift

```diff
@@ -19,6 +19,27 @@ let package = Package(
             name: "AppStoreConnect-Swift-SDK",
             dependencies: ["URLQueryEncoder"],
             path: "Sources",
-            exclude: ["OpenAPI/app_store_connect_api_2.0_openapi.json"])
+            exclude: ["OpenAPI/app_store_connect_api_2.0_openapi.json"]
+        ),
+        .binaryTarget(
+            name: "create-api",
+            url: "https://github.com/CreateAPI/CreateAPI/releases/download/0.0.5/create-api.artifactbundle.zip",
+            checksum: "89c75ec3b2938d08b961b94e70e6dd6fa0ff52a90037304d41718cd5fb58bd24"
```

Comment:
I'm inexperienced with SPM packages, would it help to document where one can get this checksum from?

Reply:
Good idea, I'll add a note! 

```suggestion
            checksum: "89c75ec3b2938d08b961b94e70e6dd6fa0ff52a90037304d41718cd5fb58bd24" // Find at https://github.com/createapi/createapi/releases/latest
```